packages/fxa-auth-server/lib/payments/paypal-client.ts

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import querystring from 'querystring';
+import superagent from 'superagent';
+import pRetry from 'p-retry';
+
+export const PAYPAL_SANDBOX_API = 'https://api-3t.sandbox.paypal.com/nvp';
+export const PAYPAL_LIVE_API = 'https://api-3t.paypal.com/nvp';
+const PAYPAL_VERSION = '204';
+const L_LIST = /L_([A-Z]+)(\d+)$/;
+
+type PaypalOptions = {
+  sandbox: boolean;
+  user: string;
+  pwd: string;
+  signature: string;
+  retryOptions?: {
+    retries?: number;
+    minTimeout?: number;
+    factor?: number;
+  };
```

Comment:
This type is similar to the one for `retryOptions` on line 38.  Could define a type T for that and this can be `Partial<T>`.